axonserver/src/main/java/io/axoniq/axonserver/AxonServerStandardAccessController.java

```diff
@@ -87,7 +76,7 @@ private Set<String> rolesForOperation(String operation) {
     }
 
     @Override
-    public Authentication authentication(String context, String token) {
+    public Authentication authentication(String token) {
```

Comment:
See below. I think `isValid` or even `authenticate` is better. Maybe too late now...